src/components/shared/SelectRecipients/SelectRecipientsDialog.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react'
+import { useRef } from 'react'
+import {
+    List,
+    makeStyles,
+    Typography,
+    Button,
+    IconButton,
+    withMobileDialog,
+    Dialog,
+    DialogTitle,
+    DialogContent,
+    DialogActions,
+} from '@material-ui/core'
+import { useStylesExtends, or } from '../../custom-ui-helper'
+import { geti18nString } from '../../../utils/i18n'
+import CloseIcon from '@material-ui/icons/Close'
+import { ProfileInList } from './ProfileInList'
+import { Profile } from '../../../database'
+import { useCurrentIdentity } from '../../DataSource/useActivatedUI'
+import { ProfileIdentifier } from '../../../database/type'
+
+const useStyles = makeStyles(theme => ({
+    dialog: {},
+    backdrop: {},
+    paper: {},
+    header: {},
```

Comment:
if you're not using those empty classes, you can add it to the withClasses declaration above.